css/index.css

```diff
@@ -1,3 +1,365 @@
+  color: white;
+}
+
+.cta img {
+  display: inline-block;
+  margin-top: 69px;
+}
+
+.main-content {
+  border-top: 1px solid black;
+  margin-top: 76px;
+  padding-top: 39px;
+}
+
+.main-content .top-content {
+  padding: 0 10px;
+}
+
+.main-content .top-content .text-content {
+  width: 410px;
+  display: inline-block;
+  margin-right: 34px;
+}
+
+.main-content .top-content .text-content:last-child {
+  margin-right: 0;
+}
+
+.middle-image {
+    width: 100%;
+    height: 182px;
+    margin-top: 45px;
+    background: url("/../img/mid-page-accent.jpg") no-repeat;
+    background-size: contain;
+}
+
+.main-content .bottom-content {
+  margin-top: 37px;
+  padding: 0 10px;
+  width: 100%;
+  display: flex;
+    justify-content: space-around;
+}
+
+.main-content .bottom-content .text-content {
+  width: 30%;
+  display: inline-block;
+  margin-right: 12px;
+
+}
+
+.main-content .bottom-content .text-content:last-child {
+  margin-right: 0;
+}
+
+.contact {
+  margin-top: 57px;
+  border-top: 1px solid black;
+  padding: 39px 10px 0;
+}
+
+.contact p {
+  margin: 20px 0;
+}
+
+footer {
+  margin: 32px 0;
+  text-align: center;
+  font-size: 14px;
+}
+
+@media screen and (max-width: 800px) {
+    .container {
+        width: 100%;
+    }
+
+    .cta img {
+        display: none;
+    }
+
+    .cta {
+        height: 25em;
+    }
+
+    .cta-text {
+        width: 100%;
+        font-size: 3em;
+    }
+
+    header {
+        display: flex;
+        flex-direction: column-reverse;
+    }
+
+    header nav {
+        width: 100%;
+    }
+
+    header .logo_container {
+        display: flex;
+        justify-content: center;
+        width: 100%;
+    }
+
+    .main-content {
+        width: 100%;
+    }
+
+    .top-content {
+        display: flex;
+    }
+
+    .top-content .text-content {
+        width: 50%;
+    }
+}
+
+@media screen and (max-width: 500px) {
+    .container {
+        width: 100%;
+    }
+
+    header {
+        display: flex;
+        flex-direction: row;
+        flex-wrap: wrap;
+        width: 100%;
+        justify-content: center;
+    }
+
+    header nav {
+        display: flex;
+        flex-wrap: wrap;
+        width: 100%;
+        margin: 0 0 3em 0;
+    }
+
+    header nav a {
+        display: flex;
+        width: 100%;
+        justify-content: center;
+        align-items: center;
+        height: 3.5em;
+        border-bottom: 1px solid black;
+        margin: 0 auto;
+    }
+
+    header .logo_container {
+        width: 100%;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        height: 3.2em;
+        order: -1;
+        border-bottom: 1px solid black;
+        margin: 0px;
+    }
+
+    .cta .cta-text {
+        display: flex;
+        width: 100%;
+        text-align: center;
+        display: inline-block;
+        vertical-align: top;
+        font-size: 3.2em;
+        text-align: center;
+        margin: 0;
+        align-content: center;
+      }
```

Comment:
Your code is pretty clean, but there a few inconsistencies with indentation. Try using prettier (https://prettier.io/).